ui/tests/acceptance/mfa-login-enforcement-test.js

```diff
@@ -0,0 +1,215 @@
+
+    await click('[data-test-mlef-cancel]');
+    assert.equal(
+      currentRouteName(),
+      'vault.cluster.access.mfa.enforcements.index',
+      'Cancel transitions to enforcements list'
+    );
+    await click('[data-test-enforcement-create]');
+    await click('.breadcrumb a');
+    assert.equal(
+      currentRouteName(),
+      'vault.cluster.access.mfa.enforcements.index',
+      'Breadcrumb transitions to enforcements list'
+    );
+    await click('[data-test-enforcement-create]');
+
+    await fillIn('[data-test-mlef-input="name"]', 'foo');
+    await click('[data-test-component="search-select"] .ember-basic-dropdown-trigger');
+    await click('.ember-power-select-option');
+    await fillIn('[data-test-mount-accessor-select]', 'auth_userpass_bb95c2b1');
+    await click('[data-test-mlef-add-target]');
+    await click('[data-test-mlef-save]');
+    assert.equal(
+      currentRouteName(),
+      'vault.cluster.access.mfa.enforcements.enforcement.index',
+      'Route transitions to enforcement on save success'
+    );
+  });
+
+  test('it should list login enforcements', async function (assert) {
+    await visit('/vault/access/mfa/enforcements');
+    assert.dom('[data-test-tab="enforcements"]').hasClass('active', 'Enforcements tab is active');
+    assert.dom('.toolbar-link').exists({ count: 1 }, 'Correct number of toolbar links render');
+    assert
+      .dom('[data-test-enforcement-create]')
+      .includesText('New enforcement', 'New enforcement link renders');
+
+    await click('[data-test-enforcement-create]');
+    assert.equal(
+      currentRouteName(),
+      'vault.cluster.access.mfa.enforcements.create',
+      'New enforcement link transitions to create route'
+    );
+    await click('[data-test-mlef-cancel]');
+
+    const enforcements = this.server.db.mfaLoginEnforcements.where({});
+    const item = enforcements[0];
+    assert.dom('[data-test-list-item]').exists({ count: enforcements.length }, 'Enforcements list renders');
+    assert
+      .dom(`[data-test-list-item="${item.name}"] svg`)
+      .hasClass('flight-icon-lock', 'Lock icon renders for list item');
+    assert.dom(`[data-test-list-item-title="${item.name}"]`).hasText(item.name, 'Enforcement name renders');
+
+    await click('[data-test-popup-menu-trigger]');
+    await click('[data-test-list-item-link="details"]');
+    assert.equal(
+      currentRouteName(),
+      'vault.cluster.access.mfa.enforcements.enforcement.index',
+      'Details more menu action transitions to enforcement route'
+    );
+    await click('.breadcrumb a');
+    await click('[data-test-popup-menu-trigger]');
+    await click('[data-test-list-item-link="edit"]');
+    assert.equal(
+      currentRouteName(),
+      'vault.cluster.access.mfa.enforcements.enforcement.edit',
+      'Edit more menu action transitions to enforcement edit route'
+    );
+  });
+
+  test('it should display login enforcement', async function (assert) {
```

Comment:
Do you need to add assert.expect(x) on these tests?

Reply:
Oops I forgot to merge main so the file change history was off. This was part of an earlier PR. I will answer the question in the other comment thought!